### ISSUE TITLE
 代码中的centernet_hourglass_coco.pth是使用voc数据集训练的

### ISSUE THREAD

**Issue by chumingqian** [bubbliiiing/centernet-pytorch#1]
您好 ：
  想确认一下， 请问注意事项中的：
   代码中的centernet_hourglass_coco.pth是使用voc数据集训练的，  到底是 coco 训练的还是 VOC 训练的

**Comment by bubbliiiing** [comment]
1、coco
2、写错字了…

**Comment by chumingqian** [comment]
还有再请问一下您，其他的pytorch版本都会把cudnn中&nbsp; batch normalization给禁掉， 您是怎么做到的



---Original---

**Comment by bubbliiiing** [comment]
batch normalization我没有禁掉
我改一下字
